src/main/java/io/cdap/plugin/google/common/GoogleAuthBaseConfig.java

```diff
@@ -158,25 +176,30 @@ private boolean validateOAuth2Properties(FailureCollector collector) {
       & checkPropertyIsSet(collector, refreshToken, REFRESH_TOKEN, REFRESH_TOKEN_LABEL);
   }
 
-  private boolean validateAccountFilePath(FailureCollector collector) {
-    if (!containsMacro(ACCOUNT_FILE_PATH)) {
+  private boolean validateServiceAccount(FailureCollector collector) {
+    if (isServiceAccountFilePath() != null && isServiceAccountFilePath()) {
```

Comment:
store the return value in a variable so you don't have to call `isServiceAccountFilePath()` twice. Same thing for `isServiceAccountJson()` below.

Reply:
PR updated.